server.go

```diff
@@ -57,16 +57,17 @@ var (
 
 // startService starts service and returns error code
 func startService() int {
-	groupsConfigPath := conf.GetGroupsConfiguration().ConfigPath
-	_, err := groups.ParseGroupConfigFile(groupsConfigPath)
+	serverCfg := conf.GetServerConfiguration()
+	groupsCfg := conf.GetGroupsConfiguration()
```

Comment:
Maybe better to move groups conf read to server `New` because as I see it `startService` should load configuration that affect on server and group configuration it's data that will be returned.

Reply:
I'm not sure about that.

It will couple HTTP server start with configuration reading...